### PR TITLE
fix docstring for Log.inverse_deriv

### PR DESCRIPTION
**Pull request by kiyoto**
fixes Issue #2306

### CODE REVIEW

**Comment by josef-pkt** [comment]
Thanks @kiyoto   

The line looks too long, it shouldn't be more than 80 characters.
Also docstring standard would require
```
name : type
    explanation
```
but large parts there don't seem to satisfy the docstring standard.

**Comment by kiyoto** [comment]
@josef-pkt thanks for the feedback. I can fix it to 80 chars/line.

>but large parts there don't seem to satisfy the docstring standard.

I can fix these, at least for `statsmodels/genmod/families/links.py`. Would it be a different PR?

**Comment by josef-pkt** [comment]
@kiyoto  You can just push to this PR, if you are willing to fix more of those. That would be very helpful.

Thanks

**Comment by kiyoto** [comment]
@josef-pkt  Got it. I already pushed the change for the 80 char issue.

Regarding the docstring standard: can you link me to a spec? I think you meant something more than https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

**Comment by josef-pkt** [comment]
docstring standard is mostly defined by numpy and used by most packages in the SciPy neighborhood
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt
section 4. and 5.
and we have many modules that have clean docstrings.

(the standard is relatively picky, because it causes sphinx warnings and failures if it's incorrect)

One extra: `__init__` of a class doesn't have a docstring, the relevant documentation goes into the class docstring.

Thanks again

**Comment by kiyoto** [comment]
@josef-pkt I think I am done for now. It would be great if you can review it.

**Comment by josef-pkt** [comment]
@kiyoto  looks good
It should render like this in html http://statsmodels.sourceforge.net/devel/generated/statsmodels.genmod.families.links.Logit.deriv.html

I'll merge it in a few minutes

Thanks

File: statsmodels/genmod/families/links.py [hunk 1]
```diff
@@ -677,7 +687,7 @@ def deriv2(self, p):
         Returns
         -------
         g'(p) : array
```
Comment (josef-pkt) [inline statsmodels/genmod/families/links.py]: this is still wrong, it should be `g''(p)`  double prime

**Comment by josef-pkt** [comment]
I will merge tomorrow morning (It's night here)

**Comment by kiyoto** [comment]
Thanks! My first contribution =)

File: statsmodels/genmod/families/links.py (hunk 1 above)
Comment (kiyoto) [inline statsmodels/genmod/families/links.py]: I just fixed this in b5f4bb71114ecb42e6434c45dc4ea2972b6cfffe

**Pull request merged**